dom/src/main/scala/org/scalajs/dom/Clipboard.scala

```diff
@@ -25,7 +25,7 @@ trait Clipboard extends EventTarget {
     *
     * To read from the clipboard, you must first have the "clipboard-read" permission.
     */
-  def read(): js.Promise[DataTransfer] = js.native
+  def read(): js.Promise[ClipboardItem] = js.native
```

Comment:
Should this be an array?
```suggestion
  def read(): js.Promise[js.Array[ClipboardItem]] = js.native
```

Reply:
Ah you're right! Didn't read MDN correctly